exercises/high-scores/canonical-data.json

```diff
@@ -103,6 +103,38 @@
           },
           "expected": [30, 50, 20, 70],
           "scenarios": ["immutable"]
+        },
+        {
+          "description": "Recent after a call to sort",
+          "property": "personalTopThree", "latest",
```

Comment:
This is not valid JSON. Is this test case still needed? I sort of assumed that the two newer ones would handle this.

Reply:
The `property` value must be a string. This looks like an array, but without the `[` and `]` characters.